### PR TITLE
 Update the external myco redis endpoint orders channel

### PR DESCRIPTION
**Pull request by Hassan754**
Adapted to changes in the [GSY-framework](https://github.com/gridsingularity/gsy-framework/pull/296)
Update the myco external matcher bids-offers endpoint

### CODE REVIEW

File: integration_tests/steps/pv_tests.py [hunk 1]
```diff
@@ -38,12 +38,12 @@ def storages_pv_final_selling_rate(context):
...
                 # Storage 1 should buy energy offers with rate more than the PV min sell rate
                 assert round(
-                    trade.offer_bid.energy_rate, DEFAULT_PRECISION) >= \
+                    trade.order.energy_rate, DEFAULT_PRECISION) >= \
                     pv.strategy.offer_update.final_rate[market.time_slot]
 
     for market in house2.past_markets:
         assert all(trade.seller == pv.name for trade in market.trades)
-        assert all(round(trade.offer_bid.energy_rate, DEFAULT_PRECISION) >=
+        assert all(round(trade.order.energy_rate, DEFAULT_PRECISION) >=
```
Comment (faizan2590) [inline integration_tests/steps/pv_tests.py:46]:
```suggestion
        assert all(limit_float_precision(trade.order.energy_rate) >=
```
Good to watch-out in rest of targeted code changes too.

Comment (Hassan754) [reply integration_tests/steps/pv_tests.py:46]: @faizan2590 https://github.com/gridsingularity/gsy-framework/pull/296#discussion_r752882210
I can update this one (it's using the default precision and it's a minor change) but if you agree, we can leave these to some time later

File: setup.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 from setuptools import find_packages, setup
 
-gsy_framework_branch = "master"
+gsy_framework_branch = "rename-bids-offers"
```
Comment (hannesdiedrich) [inline setup.py:5]: reminder to be changed before merging

File: src/gsy_e/gsy_e_core/rq_job_handler.py [hunk 3]
```diff
@@ -84,12 +84,12 @@ def launch_simulation_from_rq_job(scenario, settings, events, aggregator_device_
         config = SimulationConfig(**config_settings)
 
         spot_market_type = settings.get("spot_market_type")
-        bid_offer_match_algo = settings.get("bid_offer_match_algo")
+        orders_match_algo = settings.get("orders_match_algo")
```
Comment (hannesdiedrich) [inline src/gsy_e/gsy_e_core/rq_job_handler.py:87]: The renaming of the `bid_offer_match_algo` into `orders_match_algo` was not reflected in the gsy-web.

File: integration_tests/steps/pv_tests.py (hunk 1 above)
Comment (faizan2590) [reply integration_tests/steps/pv_tests.py:46]: As you wish but still as this is more like a refactoring PR and not targeting a feature, so still IMO this slight enhancement is worth it

**Pull request closed without merging**